msvc-full-features/JsonFormatter.vcxproj

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>

```

Comment:
Looks like the VS changes got accidentally committed again.

Reply:
Oh no, not again...